src/core/lib/surface/channel.h

```diff
@@ -145,7 +145,7 @@ class Channel : public RefCounted<Channel>,
 
   int TestOnlyRegisteredCalls() {
     MutexLock lock(&registration_table_.mu);
-    return registration_table_.map.size();
+    return static_cast<int>(registration_table_.map.size());
```

Comment:
Instead of doing a cast here, how about changing this function to return `size_t`?

Reply:
It can be done, that’ll require changes in the call stack in `ChannelTestPeer::registered_calls` and `End2endTest.ManyStubs`

Reply:
That sounds fine.

Reply:
comment addressed